### ISSUE TITLE
Update artwork

### ISSUE THREAD

**Issue by rastaman111** [yoheimuta/RxMusicPlayer#54]
Hello.  how can i update the player's artwork for example without stopping it and deleting and adding the same url?

**Comment by yoheimuta** [comment]
@rastaman111 I guess this way of updating is not supported.

**Comment by rastaman111** [comment]
@yoheimuta yes, I know that it is not supported now, but I think it is possible to implement it
